uber/models.py

```diff
@@ -944,7 +945,7 @@ def ribbon_and_or_badge(self):
 
     @property
     def is_dealer(self):
-        return bool(self.tables and (not self.registered or self.amount_paid or self.cost))
+        return bool(self.tables and self.tables != '0' and (not self.registered or self.amount_paid or self.cost))
```

Comment:
How does this get the value of ``'0'`` in the first place?  Most of the ways we parse HTTP params into model objects do type coercion.